tests/end2end/playwright/print_in_project_projection.spec.js

```diff
@@ -1,8 +1,12 @@
 // @ts-check
+import * as path from 'path';
 import { test, expect } from '@playwright/test';
-import { gotoMap } from './globals';;
+import { gotoMap, checkParameters } from './globals';;
 
-test.describe('Print in project projection', () => {
+test.describe('Print in project projection',
+    {
+      tag: ['@readonly', '@print'],
```

Comment:
We can discuss, but I was using this `@readonly` tag only if the test has been migrated to POM as well.
I know it's not linked, but it make a nice migration step ?